metricbeat/helper/metricset.go

```diff
@@ -99,3 +102,24 @@ func (m *MetricSet) createEvent(event common.MapStr) common.MapStr {
 
 	return event
 }
+
+func applySelector(event common.MapStr, selectors []string) common.MapStr {
+
+	// Not selectors set means return full events
+	if len(selectors) == 0 {
+		return event
+	}
+
+	newEvent := common.MapStr{}
+
+	for _, selector := range selectors {
+		logp.Debug("Selector", "selector, %s", selector)
```

Comment:
Aren't all debug selectors that we use today lowercase? Consider changing this to be `selector`.